packages/terra-clinical-onset-picker/docs/UPGRADEGUIDE.md

```diff
@@ -0,0 +1,31 @@
+# Terra Clinical Onset Picker Upgrade Guide
+
+## Changes from 2.0 to 3.0
+
+### Breaking Changes
+
+* New required prop `id`
+* Removed exported enums `OnsetPicker.Opts.Granularities` and `OnsetPicker.Opts.Precisions`
+* Removed default prop values for `granularity` and `precision`
```

Comment:
What do people us instead of these prop options?

Reply:
The prop is still there, it's just the default value has been removed. I suppose the default is technically undefined now.